lib/lol/request.rb

```diff
@@ -126,14 +126,14 @@ def perform_request url, verb = :get, body = nil, options = {}
       response
     end
 
-    def perform_rate_limited_request url, verb = :get, body = nil, options = {}
+    def perform_rate_limited_request (url, verb = :get, body = nil, options = {})
       return perform_uncached_request(url, verb, body, options) unless rate_limiter
       @rate_limiter.times 1 do
-        perform_uncached_request(url, verb, body, options)
+        return perform_uncached_request(url, verb, body, options)
```

Comment:
This is the main change in the file.

The rest are just cosmetic changes that the Ruby Style Guide followed by RubyMine recommended. I can revert those changes if needed. :+1: